src/org/labkey/test/tests/query/QueryMetadataTest.java

```diff
@@ -0,0 +1,210 @@
+package org.labkey.test.tests.query;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.labkey.test.BaseWebDriverTest;
+import org.labkey.test.pages.query.EditMetadataPage;
+import org.labkey.test.params.FieldDefinition;
+import org.labkey.test.params.experiment.SampleTypeDefinition;
+import org.labkey.test.params.list.IntListDefinition;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+
+@Category({})
+public class QueryMetadataTest extends BaseWebDriverTest
+{
+    static public String TEST_LIST = "queryMetadataTestList";
+    static public String TEST_SAMPLES = "queryMetadataSamples";
+
+    @Override
+    protected void doCleanup(boolean afterTest)
```

Comment:
nit: `BaseWebDriverTest` will do this cleanup for you. No need to override this method.